pkg/api/team.go

```diff
@@ -85,7 +85,7 @@ func (a *API) setupTeamRoutes() {
 
 	a.router.POST("/team/:team/edit", func(c *gin.Context) {
 		teamName := c.Param("team")
-		err := team.Update(c, a.repo, a.googleClient, a.helmClient, a.cryptor)
+		err := team.Update(c, a.repo, a.googleClient, a.helmClient, a.cryptoClient)
```

Comment:
offline-lkj

Reply:
Hvis vi skal gjøre det for jupyterhub/airflow clients så synes jeg vi også bør pakke det inn for team